### DISCUSSION TITLE
Winget release to public?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [microsoft/winget-cli#1177] [Q&A] [answered]
Hi,

When is Winget planned to release to the general public so it would install in the latest feature update or a general windows build?

Thanks,

Ward.

**Comment by denelon** [comment, marked as answer]
We're providing enterprise customers (IT Professionals) time to implement [Group Policy](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/manage-windows-package-manager-with-group-policy/ba-p/2346322).

The current target to release the Windows Package Manager with the default "App Installer" is September 2021 (not a commitment, just a planning statement).
